cmd/govim/gopls_client.go

```diff
@@ -20,30 +20,47 @@ var _ protocol.Client = (*govimplugin)(nil)
 
 func (g *govimplugin) ShowMessage(ctxt context.Context, params *protocol.ShowMessageParams) error {
 	g.logGoplsClientf("Asked to show a message: %v", params.Message)
+	if params.Type == protocol.Error {
```

Comment:
In this situation, we only want to do something if the message is an error message.

So it's probably cleaner/easier to read if we return early when we don't have an error.